client/src/pages/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/index.md

```diff
@@ -5,4 +5,12 @@ superBlock: JavaScript Algorithms and Data Structures
 ---
 ## Introduction to the Intermediate Algorithm Scripting Challenges
 
-This is a stub introduction
\ No newline at end of file
+The following 21 challenges are part of FCC's Intermediate Algorithm Scripting Challenges. These serve the purpose of preparing you before you start the final challenges to complete the Javascript Algorithms And Data Structures Certification.
```

Comment:
In case the number of challenges decreases or increase, just remove the 21.  Also, see my suggested change for the second sentence, to keep prevent so many prepositional phrases (3).
```suggestion
The following challenges are part of FCC's Intermediate Algorithm Scripting Challenges. These should prepare you to complete the final challenges, which will complete the Javascript Algorithms And Data Structures Certification.
```

Reply:
```suggestion
The following challenges are part of FCC's Intermediate Algorithm Scripting Challenges. These should prepare you to complete the final challenges for the JavaScript Algorithms And Data Structures Certification.
```
How does this sound?

Reply:
I like your latest suggestion.  I tried to commit the changes, but you have not given maintainers permission to make changes to your branch for this PR.

Reply:
This box is turned on:
![image](https://user-images.githubusercontent.com/34807532/54137811-54a3da00-4444-11e9-8f4e-760640027252.png)

Reply:
Strange.  It will not let me commit the changes.  I just tried again.

![image](https://user-images.githubusercontent.com/5313213/54137989-e3c8d780-43db-11e9-9535-149726ae91a0.png)

Can you just make the changes, so that I can review the final changes?

Reply:
Alright, one sec